extensions/depthmap2mask/install.py

```diff
@@ -0,0 +1,69 @@
+# import subprocess
```

Comment:
I don't want to remove this yet from the `stable-diffusion-extensions` repo.  I am waiting until we can verify that the SD Web UI works.